Library/Homebrew/software_spec.rb

```diff
@@ -444,18 +444,11 @@ def prefix=(prefix)
 
   def root_url(var = nil, specs = {})
     if var.nil?
-      @root_url ||= if Homebrew::EnvConfig.bottle_domain.match?(GitHubPackages::URL_REGEX)
-        GitHubPackages.root_url(tap.user, tap.repo).to_s
-      else
-        "#{Homebrew::EnvConfig.bottle_domain}/#{Utils::Bottles::Bintray.repository(tap)}"
-      end
+      @root_url ||= GitHubPackages.sanitize_root_url(
+        "#{Homebrew::EnvConfig.bottle_domain}/#{Utils::Bottles::Bintray.repository(tap)}",
```

Comment:
I’d rather we didn’t reference bintray here. It’s just going to make it harder to remove that code later.